rust-toolchain.toml

```diff
@@ -0,0 +1,10 @@
+[toolchain]
+# Pinned due to https://github.com/rust-osdev/uefi-rs/issues/500.
+#
+# Compilation started failing 2022-09-02. The previous several builds
+# don't have miri available though, so pin to a slightly older version.
+channel = "nightly-2022-08-26"
+components = [
+    # Needed for `-Zbuild-std`.
```

Comment:
very minor nit: probably you can add a comment here

"additive components to the default profile" or so

Feel free to merge with or without this!

Reply:
:+1: added